src/pages/map.tsx

```diff
@@ -165,135 +166,141 @@ const MapPage = ({ schedule }: Props) => {
   const jakartaLatLng = { lat: -6.163088, lng: 106.836715 };
 
   return (
-    <Container minHeight="100vh">
-      <Map
-        containerStyle={{
-          height: '100vh',
-          width: '100%'
-        }}
-        onDrag={() => setActiveLoc(undefined)}
-        onStyleLoad={loadedMap => {
-          setMap(loadedMap);
-          loadedMap.setCenter(jakartaLatLng);
-          setInitialMapBound();
-        }}
-        style="mapbox://styles/mapbox/streets-v8"
-      >
-        <>
-          {coordinates.map((coordinate, i) => {
-            return (
-              //@ts-expect-error - coordinate type conflict
-              <Marker key={i} coordinates={coordinate}>
-                <Box
-                  onClick={() => {
-                    setActiveLoc(coordinate.lokasi);
-                    if (map != null) {
-                      map.easeTo({
+    <>
+      <Head>
+        <title>Lokasi dan Jadwal Vaksinasi DKI Jakarta</title>
+      </Head>
```

Comment:
`Head` works on any wrapping component. No need to wrap the entire component again inside a Fragment.